bedrock/base/templates/404.html

```diff
@@ -13,6 +15,18 @@
   {% stylesheet 'page_not_found' %}
 {% endblock %}
 
+{% block js %}
+  {% if LANG.startswith('en-') and switch('mozorg-search-general') %}
+    {% javascript 'custom-search' %}
+  {% endif %}
+{% endblock %}
+
+{% block site_header_unwrapped %}
+  {% if LANG.startswith('en-') and switch('mozorg-search-general') %}
+    {{ google_custom_search(settings.GOOG_CUSTOM_SEARCH_TYPES['general'], 'moz-search-general', _('Internet Health'), _('Volunteer, Events, Add-ons, Firefox, Internet Health')) }}
```

Comment:
oh, yeah, I suppose that would be better... but this is probably temporary anyway, probably to be removed in a few weeks.